<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSSceneCollections</class>
 <widget class="QDialog" name="OBSSceneCollections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SceneCollections.Title</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="buttonBulkMode">
         <property name="icon">
          <iconset>
           <normaloff>:/res/images/check_round_outline.svg</normaloff>:/res/images/check_round_outline.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="class" stdset="0">
          <string notr="true">icon-check-round-outline</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineeditSearch">
       <property name="placeholderText">
        <string>SceneCollections.Search</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSort">
         <property name="icon">
          <iconset>
           <normaloff>:/res/images/vertical_arrows_updown.svg</normaloff>:/res/images/vertical_arrows_updown.svg</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="class" stdset="0">
          <string notr="true">icon-vertical-arrows-updown</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget">
      <layout class="QVBoxLayout">
       <property name="alignment">
        <set>Qt::AlignTop|Qt::AlignHCenter</set>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="idian::Group" name="group">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>42</width>
           <height>42</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonNew">
       <property name="text">
        <string>SceneCollections.New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonImport">
       <property name="text">
        <string>SceneCollections.Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExportBulk">
       <property name="text">
        <string>SceneCollections.Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDuplicateBulk">
       <property name="text">
        <string>SceneCollections.Duplicate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeleteBulk">
       <property name="text">
        <string>SceneCollections.Delete</string>
       </property>
       <property name="class" stdset="0">
        <string notr="true">bg-danger</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>idian::Group</class>
   <extends>QFrame</extends>
   <header>Idian/Idian.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
